src/ui-lib/sass/05-components/01-atoms/06-buttons/_08-toggle-menu.scss

```diff
@@ -1,31 +1,105 @@
+$grav-sp-icon: 0.35rem;
+$grav-tr-icon-hover: 0.1rem;
+$grav-tr-icon-rotate: 135deg;
+$grav-tr-icon-scale: 1.2;
+$grav-size-icon: 33px;
+
 .grav-c-toggle-menu {
+  display: flex;
+  height: $grav-size-icon;
   margin-top: 0;
-  border: $grav-st-thickness solid $grav-co-neutral-asphalt;
+  border: hidden;
   background-color: inherit;
-  color: $grav-co-bg;
-
-  &:hover {
+  text-align: center;
+  
+  &--inverted,
+  .is-light & {
+    &__icon,
+    &__icon::before,
+    &__icon::after {
+      background-color: $grav-co-neutral-asphalt;
+    }
+     
+    &[aria-pressed='true']:hover,
+    &[aria-pressed='true']:active,
+    &[aria-pressed='true']:focus {
+      background-color: inherit;
+    }
+  }
+  
+  &:hover,
+  &:active,
+  &:focus {
```

Comment:
Don't need this block either :-)

(No need to set the cursor since the `<button>` element already make browsers show a suitable cursor when hovering over them)